### DISCUSSION TITLE
Set of pre-created prompt templates for new users

### DISCUSSION THREAD

**Discussion by lcgcastro96** [danny-avila/LibreChat#4478] [Q&A]
Hello everyone! 

I'm interested in trying to create a list of pre-written prompt templates for any user who logs in into LibreChat. I've considered trying to automatize a command that directly populates MongoDB's templates for each user, but I'm finding it not being a clever solution.

Has anyone done something similar? Is this possible at all?

Thanks a lot for this work, LibreChat is amazing! Keep the good work!

**Comment by danny-avila** [comment]
A way of importing prompts and marking them as global in the csv import could be a good way of doing this, would need to be implemented.
